Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -294,7 +295,10 @@ private bool CanReadAll(string resource)
                                       // our API.
                 case Resource.ADVISOR:
                     // User is admin
```

Comment:
We should change this comment to something like:
```
// User is authorized to view academic info
```
Since site admins are no longer authorized.

Reply:
Sounds good! I changed the comment.

Also, to the question above, yes I did, but I think @EjPlatzer and I eventually decided to remove it. From what I remember, it was because AuthorGroup Advisors was going to be removed because it didn't actually need to be used anywhere and if I had used it here it would have complicated the work afterwards. The code we did elsewhere would have done the job of removing the AdvisorID from the student profile, so we didn't need to use it here.

Reply:
No problem. I was wondering if that would be a better way to approach this.

Best practice is to create an issue for any future work that you are hoping to accomplish (or you are hoping someone else may accomplish) so that we don't forget about it. Especially when it relates to something you're choosing not to do now. Have you created an issue for removing AdvisorIDs from the student profile object on the project board? If not, it's probably a good opportunity for you to do so!

Reply:
I'm not sure how to create an issue but I will try! Maybe we can also discuss this in your office hour tomorrow.